cherryontech/jupiter-jumpers
# Profile 2

## Description
I updated profile page according to Figma 

## Related Issue

https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-57
## Related Story Card



updated the profile page 

## Acceptance Criteria

Given I have clicked on my “Account” in the user dropdown

When I view my profile details

Then I have the option to modify existing information: name, avatar, password, career status.


<img width="1101" alt="Screenshot 2024-06-07 at 7 07 34 PM" src="https://github.com/cherryontech/Uncloud/assets/123599985/76fea7de-2de7-4512-b460-ddd41da3999f">

## Testing Instructions

1. Ensure you are logged in and navigate to the profile page.
2. Verify user data (name, gender, career status, location, profile image) loads correctly.
3. Change the name, gender, career status, and location fields.
4. Upload a new profile image and verify the preview updates, then remove the image to see the default.
5. Click "Save Changes" and check for a success message, ensuring data updates correctly.
6. Click "Cancel" after making changes to verify the form resets.
